### PR TITLE
samples: add XBee Bluetooth samples

### PR DESCRIPTION
**Pull request by diescalo**
Signed-off-by: Diego Escalona <diego.escalona@digi.com>

### CODE REVIEW

File: samples/xbee/communication/bluetooth/ReceiveBluetoothDataSample/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+* An Android or iOS device with the Digi XBee Mobile application installed
+  (available at Google Play and App Store).
+
+Setup
+-----
+1. Enable the XBee Bluetooth interface of the XBee Gateway. To do so:
+   
+   1. Open the dashboard page of the device and go to 'System > Device
+      Configuration'.
+   2. Find and expand the 'XBee' configuration group
```
Comment (rubenmoral) [inline samples/xbee/communication/bluetooth/ReceiveBluetoothDataSample/README.md]: NP: missing final period.

File: samples/xbee/communication/bluetooth/ReceiveBluetoothDataSample/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+Receive Bluetooth Data Sample Application
+=========================================
+
+This sample Python application shows how to receive and process data coming
+from the Bluetooth interface of the XBee device.
```
Comment (rubenmoral) [inline samples/xbee/communication/bluetooth/ReceiveBluetoothDataSample/README.md]: Should we refer to 'XBee device' or 'XBee Gateway'?

File: samples/xbee/communication/bluetooth/ReceiveBluetoothDataSample/README.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  (available at Google Play and App Store).
+
+Setup
+-----
+1. Enable the XBee Bluetooth interface of the XBee Gateway. To do so:
+   
+   1. Open the dashboard page of the device and go to 'System > Device
+      Configuration'.
+   2. Find and expand the 'XBee' configuration group
+   3. Enable the "Bluetooth" interface.
```
Comment (rubenmoral) [inline samples/xbee/communication/bluetooth/ReceiveBluetoothDataSample/README.md]: NP: use single quotation marks to keep consistency.

File: samples/xbee/communication/bluetooth/SendBluetoothDataSample/README.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
+Send Bluetooth Data Sample Application
```
Comment (rubenmoral) [inline samples/xbee/communication/bluetooth/SendBluetoothDataSample/README.md:1]: Same comments than in the previous README.

**Review by rubenmoral — APPROVED** [review]

**Review by hgonzaleDigi — APPROVED** [review]

**Pull request merged**